pkg/command/cluster/clusteradd/cmd.go

```diff
@@ -85,6 +87,7 @@ func (cmd *command) run() error {
 	c := &managerclient.Cluster{
 		ID:                     cmd.id,
 		Name:                   cmd.name,
+		Labels:                 cmd.label.ApplyDiff(nil),
```

Comment:
Why not to create method ToMap() on label directly ?
I think it's more intuitive than applying a diff of nil to convert Label to map.